src/styles/zoomwindow.scss

```diff
@@ -0,0 +1,53 @@
+@import "variables.scss";
+
+.zoomwindow {
+  background: white;
+  position: fixed;
+  left: 5px;
+  top: 5px;
+  z-index: 1;
+  width: 25px;
+  height: 25px;
+  padding-right:40px;
+  padding-left:5px;
+  align-content: center;
+  border-width: 3px;
+  border-style: solid;
+  border-color: $navbar-color;
+  margin: 3px;
+  margin-top: 50px;
+
+  &--hide {
+   opacity: 0;
+   animation-name: fadeOutOpacity;
```

Comment:
`opacity` can be handled using CSS `transition`. using `animation` is overkill. if you use a `transition`, you don't need to define keyframes at the bottom

example: https://stackoverflow.com/a/12992210/5773182 (but set the transition on both to make it fade in and out)